crates/ruff_server/src/server/api/requests/code_action.rs

```diff
@@ -1,81 +1,129 @@
-use crate::edit::ToRangeExt;
+use crate::lint::fixes_for_diagnostics;
 use crate::server::api::LSPResult;
+use crate::server::SupportedCodeActionKind;
 use crate::server::{client::Notifier, Result};
 use crate::session::DocumentSnapshot;
+use crate::DIAGNOSTIC_NAME;
+use lsp_server::ErrorCode;
 use lsp_types::{self as types, request as req};
-use ruff_text_size::Ranged;
+use rustc_hash::FxHashSet;
+use types::{CodeActionKind, CodeActionOrCommand};
 
-pub(crate) struct CodeAction;
+use super::code_action_resolve::resolve_edit_for_fix_all;
 
-impl super::RequestHandler for CodeAction {
+pub(crate) struct CodeActions;
+
+impl super::RequestHandler for CodeActions {
     type RequestType = req::CodeActionRequest;
 }
 
-impl super::BackgroundDocumentRequestHandler for CodeAction {
+impl super::BackgroundDocumentRequestHandler for CodeActions {
     super::define_document_url!(params: &types::CodeActionParams);
     fn run_with_snapshot(
         snapshot: DocumentSnapshot,
         _notifier: Notifier,
         params: types::CodeActionParams,
     ) -> Result<Option<types::CodeActionResponse>> {
+        let mut response: types::CodeActionResponse = types::CodeActionResponse::default();
+
+        let supported_code_actions = supported_code_actions(params.context.only);
+
+        tracing::error!("{supported_code_actions:?}");
```

Comment:
Did this get accidentally checked-in or is it suppose to be at a different log level?

Reply:
This was checked in by accident, my bad.